pkgs/by-name/im/immich-cli/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  immich,
+  buildNpmPackage,
+  nodejs,
+  makeWrapper,
+}:
+buildNpmPackage {
+  pname = "immich-cli";
+  src = "${immich.src}/cli";
+  inherit (immich.sources.components.cli) version npmDepsHash;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  inherit (immich.web) preBuild;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    mv package.json package-lock.json node_modules dist $out/
+
+    ls $out/dist
+
+    makeWrapper ${lib.getExe nodejs} $out/bin/immich --add-flags $out/dist/index.js
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Self-hosted photo and video backup solution (command line interface)";
+    homepage = "https://immich.app/";
```

I'm not sure what the convention is in nixpkgs for this, but the following could be neat. I'm mixed on this because the current approach keeps things consistent across all Immich components
```suggestion
    homepage = "https://immich.app/docs/features/command-line-interface";
```
Would appreciate someone else confirming/denying this.